src/mono/mono/mini/CMakeLists.txt

```diff
@@ -658,3 +658,6 @@ if(NOT DISABLE_EXECUTABLES)
 
   install_with_stripped_symbols(mono-sgen TARGETS bin)
 endif()
+configure_file(
+  ${CLR_SRC_NATIVE_DIR}/libs/Common/pal_config.h.in
+  ${CMAKE_CURRENT_BINARY_DIR}/pal_config.h)
```

Comment:
we shouldn't mix our configs with pal_config.h.in, if you need any of the `HAVE_*` defines then you need to include them in src/mono/cmake/configure.cmake

Reply:
I want to use `assert_msg` from https://github.com/mkhamoyan/runtime/blob/main/src/native/libs/Common/pal_utilities.h#L27 in `pal_placeholders.c` (instead of defining them again here https://github.com/mkhamoyan/runtime/blob/main/src/native/libs/System.Globalization.Native/pal_placeholders.c#L18-L38) . But when including `"pal_utilities.h"` header  in `pal_placeholders.c`, it complains that can't find `pal_config.h`.

Reply:
You can either move it to a new header file in src/native/minipal since that one is supposed to be shared between runtimes.

Or given our future direction of wanting to stop compiling globalization in mono maybe it's better to just copy the `assert_msg` definition into mono since it will get removed later anyway.

Reply:
Thanks, I will keep initial way of copying `assert_msg` definition into mono.